tests/test_core.py

```diff
@@ -264,36 +265,36 @@ def prepare_download_entry(req, tmpdir, format_map=core.EFormats, human_readable
 
 def test_download_entry_genbank(req, tmpdir):
     entry, outdir, joblist = prepare_download_entry(req, tmpdir)
-    jobs = core.download_entry(entry, 'refseq', 'bacteria', str(outdir), 'genbank', None)
+    jobs = core.download_entry(entry, 'refseq', 'bacteria', str(outdir), ['genbank'], None)
     expected = [j for j in joblist if j.local_file.endswith('_genomic.gbff.gz')]
     assert jobs == expected
 
 
 def test_download_entry_all(req, tmpdir):
     entry, outdir, expected = prepare_download_entry(req, tmpdir)
-    jobs = core.download_entry(entry, 'refseq', 'bacteria', str(outdir), 'all', None)
+    jobs = core.download_entry(entry, 'refseq', 'bacteria', str(outdir), core.EFormats.keys(), None)
```

Comment:
This is removing the check for the 'all' logic, isn't it?

Reply:
The all logic is moved outside the download_entry method.
Download_entry now only supports a list of formats.
I will rewrite the test so it checks the all logic.

Reply:
Correction: did not rewrite the test. But added a new test